### PR TITLE
Update API docs after new endpoint was added

### PR DESCRIPTION
**Pull request by sonjaer**
# Hey, I just made a Pull Request!

## Description
Update API docs after new endpoint was added

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+workflows
```
Comment (sonjaer) [inline .circleci/config.yml:2]:
```suggestion
```

File: .circleci/config.yml
Comment (narape) [inline .circleci/config.yml:1]: What is this file doing? why is needed? and why a no-op job?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: 2.1
+jobs:
+  no-op:
+    docker:
+      - image: circleci/python:3.9
```
Comment (sonjaer) [inline .circleci/config.yml:5]:
```suggestion
      - image: non-existing-image
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+version: 2.1
+jobs:
+  no-op:
+    docker:
+      - image: non-existing-image
```
Comment (sonjaer) [inline .circleci/config.yml:5]:
```suggestion
      - image: circleci/python:3.9
```

File: .circleci/config.yml
Comment (sonjaer) [reply .circleci/config.yml:1]: this is for github pages to show api html doc, and we don't care about running the build here

circle ci complains if the file is missing or empty so this is the only way I found works

**Review by narape — APPROVED** [review]

**Pull request merged**